docs/user-guide/sinks/fallback.md

```diff
@@ -49,3 +55,12 @@ User Defined Sink as a fallback sink:
             container:
               image: my-sink:latest
 ```
+### Step 2 - update the user-defined sink implementation
+
+Make code changes in the primary sink to return either a **failed** response or a **fallback** response.
+
+* a **failed** response gets processed following the [retry strategy](https://numaflow.numaproj.io/user-guide/sinks/retry-strategy/), and if the retry strategy is set to `fallback`, the message will be directed to the fallback sink after the retries are exhausted.
+* a **fallback** response doesn't respect the sink retry strategy. It gets immediately directed to the fallback sink without getting retried.
+
+SDKs to generate a fallback/failed response in a primary user-defined sink:
```

Comment:
```suggestion
SDKs has to generate a fallback/failed response (e.g., in Java user has to return `responseFallback(String id)`) in the primary user-defined sink:
```

Reply:
SDKs have to generate a fallback/failed response. (e.g., in Java user has to return `responseFallback(String id)`) in the primary user-defined sink: